pwiz_tools/Skyline/Model/DocSettings/SrmSettings.cs

```diff
@@ -2578,7 +2578,7 @@ public SrmSettingsDiff(SrmSettings settingsOld, SrmSettings settingsNew)
             // changes, or if constraint is non-None and background proteome or digestion enzyme changes
             // Background proteome uniqueness constraints - only considered a change if constraint type
             // changes, or if constraint is non-None and background proteome or digestion enzyme changes
-            bool uniquenessConstraintChange = !Equals(newPep.Filter.PeptideUniqueness, oldPep.Filter.PeptideUniqueness);
+            bool uniquenessConstraintChange = uniquenessConstraintChange = !Equals(newPep.Filter.PeptideUniqueness, oldPep.Filter.PeptideUniqueness);

```

Comment:
this seems weird
uniquenessConstraintChange = uniquenessConstraintChange =

Reply:
Oops. That's a weird line of code for me to have written. There was supposed to be a "&&" in there instead of the "=".